packages/unpublished-fastboot-test-app/tsconfig.json

```diff
@@ -1,11 +1,34 @@
 {
   "extends": "../../tsconfig.json",
+  "baseUrl": "../..",
   "compilerOptions": {
     "paths": {
-      "fastboot-test-app/tests/*": ["tests/*"],
-      "fastboot-test-app/*": ["app/*"],
-      "*": ["types/*"]
+      "ember-data": ["packages/-ember-data/addon"],
```

Comment:
why do we need to set these? does extending clobber vs merge?

Reply:
For reference it does not appear that it should clobber:

```
The extends is a top-level property in tsconfig.json (alongside compilerOptions, files, include, and exclude). extends’ value is a string containing a path to another configuration file to inherit from.

The configuration from the base file are loaded first, then overridden by those in the inheriting config file. If a circularity is encountered, we report an error.

files, include and exclude from the inheriting config file overwrite those from the base config file.

All relative paths found in the configuration file will be resolved relative to the configuration file they originated in.
```

https://www.typescriptlang.org/docs/handbook/tsconfig-json.html

Reply:
See https://github.com/microsoft/TypeScript/issues/14527